onnxruntime/core/mlas/lib/sve/elementwise_sve.cpp

```diff
@@ -0,0 +1,685 @@
+/*++
+
+Copyright 2025 FUJITSU LIMITED
+
+Module Name:
+
+    elementwise_sve.cpp
+
+Abstract:
+
+    This module contains the implementation of SVE-based elementwise operations
+--*/
+
+#ifdef __ARM_FEATURE_SVE
```

Comment:
Is this guard still needed with all the cmake changes that guard the compilation of the SVE files ?